wordpress-mobile/WordPress-Android
# Page List : Support for Unhandled Version Conflict

Fixes #11277
## Findings
With this solution, the goal was to add the unhandled version conflict to the Pages List. However, upon doing the implementation it was realized that the `PageStore` and it's associated functions that support fetching pages don't support updating the local `Posts` with the last modified date of the remote post when an update has been done on another client. So this lead to the exploration of adding this functionality to [Flux C in this PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1504). More details will be in the comments there. Until a proper solution has been arrived at, this PR will be a draft. 

## Solution
Add a dialog that allows the user to resolve a page conflict when changes were made on another device and there are local changes. 

## Testing
1. Make changes to a post on your device, but don't save, just press back so it's locally saved. 
2. Make changes to the same post on Calypso and press update. 
3. A `version conflict` label should be shown and when you attempt to open the page, then a dialog would allow you to choose which version of the post you want to keep. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
